pom.xml

```diff
@@ -400,6 +400,11 @@
 			<version>1.5.3</version>
 			<scope>test</scope>
 		</dependency>
+		 <dependency>
```

Comment:
If you push the changes to the same branch (as you did) it will update this PR. No need for an additional PR.